scripts/gc.sh

```diff
@@ -56,17 +54,21 @@ import { buildProps } from '@element-plus/utils'
 import type { ExtractPropTypes } from 'vue'
 import type $NAME from './$INPUT_NAME.vue'
 
-export const ${INPUT_NAME}Props = buildProps({})
+export const ${PROP_NAME}Props = buildProps({} as const)
+export type ${NAME}Props = ExtractPropTypes<typeof ${PROP_NAME}Props>
+
+export const ${PROP_NAME}Emits = {}
+export type ${NAME}Emits = typeof ${PROP_NAME}Emits
```

Comment:
Regarding the following instance, do we need to add an `instance.ts` file like the button component?